suite-native/graph/src/components/GraphError.tsx

```diff
@@ -36,23 +38,26 @@ const ErrorIcon = () => {
 
 export const GraphError = ({ error, onTryAgain }: GraphErrorProps) => {
     const { applyStyle } = useNativeStyles();
+    const { translate } = useTranslate();
 
     return (
         <VStack spacing="small" alignItems="center" paddingHorizontal="medium">
             <ErrorIcon />
             <Animated.View entering={FadeInDown} exiting={FadeOutUp}>
                 <Text variant="hint" color="textSubdued" textAlign="center">
-                    There are some troubles with loading graph points: {error}
+                    {translate('graph.errorMessage')}
+                    {error}
                 </Text>
-                <Text.Pressable
```

Comment:
What's wrong with this pressable?

Reply:
It's not propagating the event and the `onPress` is not working somehow. I don't have a clue why, but it's the only usage so I guess wrapping with `Pressable` is pretty good, since the `Text` is still going to be performant, just wrapped.

Reply:
If it's not working we should remove that Text.Pressable completely

Reply:
https://github.com/trezor/trezor-suite/pull/11495/commits/a35c02f7b15e79085b5910adb9628e61425302ce